### PR TITLE
add Grunt and ESLint support

### PR DESCRIPTION
**Pull request by danbev**
Motivation:
Introduce ESLint to this project and in the process make the build
process of our modules consistent we want to introduce
Grunt.

Modifications:
Added Grunt.js and the required dev dependencies.
Added grunt to be run by Travis (CI).

Result:
Running grunt will run eslint

JIRA:
https://issues.jboss.org/browse/RAINCATCH-149

### CODE REVIEW

**Review by nialldonnellyfh — APPROVED** [review]
Looks good

**Review by nialldonnellyfh — APPROVED** [review]
Looks good.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "lib/angular/vehicle-inspection-ng.js",
   "scripts": {
     "build": "wfm-template-build -m 'wfm.vehicle-inspection'",
-    "watch": "wfm-template-build -w -m 'wfm.vehicle-inspection'"
+    "watch": "wfm-template-build -w -m 'wfm.vehicle-inspection'",
+    "test": "grunt"
```
Comment (evanshortiss) [inline package.json:9]: This popped into my feed, 2 cents to add. No need to add grunt and extra deps, just add eslint to devDeps and maybe do:

```
"test": "eslint lib/*.js lib/**/*.js"
```

Enjoy!

Comment (nialldonnellyfh) [reply package.json:9]: Hi @evanshortiss . Haven't talked in ages!! Thanks for the feedback.

We are introducing grunt to the raincatcher components to get a more accessible build system. The eslint job is just the start, but eventually all of the build tasks will be done through grunt.

**Pull request merged**